mage_ai/data_preparation/templates/transformers/transformer_actions/column/max.py

```diff
@@ -0,0 +1,29 @@
+from mage_ai.data_cleaner.transformer_actions.base import BaseAction
+from mage_ai.data_cleaner.transformer_actions.constants import ActionType, Axis
+from mage_ai.data_cleaner.transformer_actions.utils import build_transformer_action
+from pandas import DataFrame
+
+if 'transformer' not in globals():
+    from mage_ai.data_preparation.decorators import transformer
+
+
+@transformer
+def execute_transformer_action(df: DataFrame, *args, **kwargs) -> DataFrame:
+    """
+    Execute Transformer Action: ActionType.MAX
+    """
+    action = build_transformer_action(
+        df,
+        action_type=ActionType.MAX,
+        action_code='',  # Enter filtering condition on rows before aggregation
+        arguments=[],  # Enter the columns to compute aggregate over
+        axis=Axis.COLUMN,
+        options={'groupby_columns': []},  # Enter columns to group by
+        outputs=[
+            # The number of outputs below must match the number of arguments
+            {'uuid': 'new_aggregate_column_1', 'column_type': 'category'},
```

Comment:
let's only specify number column type for max, min, median. this aggregation is usually used for numbers.

Reply:
Switched output types to `number_with_decimals` to indicate inclusion floating point numbers.